lesson3/task3.js

```diff
@@ -0,0 +1,44 @@
+let word = "ЗдесьЯКОБЫ текст ДЛЯ проверки__9";
+let glasnie = ["а", "о", "и", "е", "ё", "э", "ы", "у", "ю", "я"];
+let soglasnie = [
+  "б",
+  "в",
+  "г",
+  "д",
+  "ж",
+  "з",
+  "й",
+  "к",
+  "л",
+  "м",
+  "н",
+  "п",
+  "р",
+  "с",
+  "т",
+  "ф",
+  "х",
+  "ц",
+  "ч",
+  "ш",
+  "щ",
+  "ь",
+  "ъ",
+];
+let glasnieScore = 0;
+let soglasnieScore = 0;
+let changedRegistr = word.toLowerCase();
+
+for (i = 0; i <= changedRegistr.length; i++) {
+  let box = changedRegistr[i];
+  if (glasnie.includes(box)) {
+    glasnieScore += 1;
+  }
+  if (soglasnie.includes(box)) {
+    soglasnieScore += 1;
+  }
+}
```

Comment:
Не забьіваем про более подходящее именование переменньіх, упрощает читаемость в разьі
```suggestion
const lowerCasedStr = word.toLowerCase();

for (let i = 0; i <= lowerCasedStr.length; i++) {
  const char = lowerCasedStr[i];
  if (glasnie.includes(char)) {
    glasnieScore += 1;
  }
  if (soglasnie.includes(char)) {
    soglasnieScore += 1;
  }
}
```